rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1855,14 +1855,17 @@ private void logReleaseBtc(BtcTransaction btcTx, byte[] releaseCreationRskTxHash
 
     private void processSigning(
         FederationMember federatorMember,
+        Federation federation,
         List<byte[]> signatures,
         Keccak256 releaseCreationRskTxHash,
-        BtcTransaction btcTx) {
+        BtcTransaction btcTx
+    ) {
 
-        // Build input hashes for signatures
+        int federationFormatVersion = federation.getFormatVersion();
+        // Build input sig hashes for signatures
         List<Sha256Hash> sigHashes = new ArrayList<>();
         for (int i = 0; i < btcTx.getInputs().size(); i++) {
-            Sha256Hash sigHash = generateSigHashForP2SHTransactionInput(btcTx, i);
+            Sha256Hash sigHash = generateSigHashForFederationFormatVersion(btcTx, i, federationFormatVersion);
             sigHashes.add(sigHash);
         }
```

Comment:
Should all this be in BitcoinUtils maybe? Get list of sig hashes for all inputs of a transaction